spec/system/show_terms_of_deposit_spec.rb

```diff
@@ -12,6 +12,9 @@
 
     click_link_or_button 'Terms of Deposit'
 
+    # The lazy loading of the modal fails sporadically without this sleep.
+    sleep 0.25
```

Comment:
Could we replace this with a `find` call that, e.g., looks for unique text in the modal? Then we can rely on Capybara's built-in waiting ability instead of a hard-coded sleep value.

Reply:
You'd think it would. I tried multiple approaches (looking for text, looking for the completed attribute on the turbo frame) but none would consistently work.